raiden/network/rpc/client.py

```diff
@@ -217,12 +224,10 @@ def gaslimit(self, location='latest') -> int:
         gas_limit = self.web3.eth.getBlock(location)['gasLimit']
         return gas_limit * 8 // 10
```

Comment:
Any chance of also including better gas limit calculations on this PR? =)

Reply:
No, not that I know of. What do you mean exactly?

Reply:
Mh, seems like `gaslimit` is only used in the assumption tests anyways. we can probably remove that all together.